api/services/elastigroup/azure/spotVms/paths/vmProtection.yaml

```diff
@@ -53,8 +54,9 @@ delete:
   tags:
     - Elastigroup Azure Spot VMs
   parameters:
+    - $ref: "../../../../../commons/parameters/accountId.yaml"
```

Comment:
added in the previous PR.
please align your branch with the most up-to-date master